### PR TITLE
fix: (core) stackblitz files for dropdown

### PR DESCRIPTION
**Pull request by stefanoScalzo**
#### Please provide a link to the associated issue.
fixes: #1595
#### Please provide a brief summary of this pull request.
Fixing the typescript files so they can work with dropdowns
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -1,228 +1,260 @@
...
-    "projects": {
-        "docs": {
-            "root": "apps/docs/",
-            "sourceRoot": "apps/docs/src",
-            "projectType": "application",
-            "prefix": "fundamental-ngx",
-            "schematics": {
-                "@nrwl/schematics:component": {
-                    "style": "scss"
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```
Comment (droshev) [inline angular.json]: did you change this file intentionally?

**Comment by droshev** [comment]
@stefanoScalzo you need to change the title of the PR

File: angular.json (hunk 1 above)
Comment (stefanoScalzo) [reply angular.json]: no i will revert the changes

**Comment by JKMarkowski** [comment]
@stefanoScalzo  Besides title and this `angular.json` it looks good to me.

**Pull request merged**